tools/cppcheck/suppressions.xml

```diff
@@ -21,6 +21,18 @@
         <fileName>*/src/common/src/convert.cc</fileName>
     </suppress>
 
+    <!-- Suppress unused ctc callback handler functions -->
+    <suppress>
+        <id>unusedFunction</id>
+        <fileName>*/src/modules/ctc/src/ctc_callback_handler.cc</fileName>
+    </suppress>
+
+    <!-- Suppress useStlAlgorithm suggestion ctc callback handler -->
```

Comment:
Why is this being suppressed?

Reply:
it suggested to use back insert because I'm using for loop to populate `StandardTableView` component on UI, but that won't work here with slint object:
![image](https://github.com/user-attachments/assets/8637c548-00b4-41bb-94ff-7e13edf70509)
- file: src/modules/ctc/src/ctc_callback_handler.cc 
- line: 42